added python getting started devlab (#2)
Snowflake-Labs/devlabs: site/devlabs/getting_started_with_python.md

summary: Getting Started with Python
id: getting_started_with_python

> categories: 

Reviewer: ```suggestion
categories: data-applications
```

tags: python
status: Draft

> Feedback Link: https://developers.snowflake.com

Reviewer: ```suggestion
Feedback Link: https://github.com/Snowflake-Labs/devlabs/issues
```

# Getting Started with Python
<!-- ------------------------ -->

> ## Interact with Snowflake using Python
> Duration: 1
> 
> You can connect to Snowflake in many languages. If your language of choice is Python, you'll want to begin here to connect to Snowflake. We'll walk you through getting the Python Connector up and running, and then explore the basic operations you can do with it. You'll find the Python Connector to be quite robust, as it even supports integration with Pandas DataFrames.
> 
> After following this guide, you'll be able to delve into the rest of our Python Connector documentation and fully flesh out your Snowflake connected Python applications. Let's begin!
> 

Reviewer: This is looking really good. Please add the overview sections as the first step: prereq, what you'll learn/need/build. I added an outline from the sample as a suggested edit. Please update the suggestion with the correct content. 

```suggestion
## Overview
Duration: 1

You can connect to Snowflake in many languages. If your language of choice is Python, you'll want to begin here to connect to Snowflake. We'll walk you through getting the Python Connector up and running, and then explore the basic operations you can do with it. You'll find the Python Connector to be quite robust, as it even supports integration with Pandas DataFrames.

### Prerequisites
- something

### What You’ll Learn 
- how to set the metadata for a guide (category, author, id, etc)
- how to set the amount of time each slide will take to finish 
- how to include code snippets 
- how to hyperlink items 
- how to include images 

### What You’ll Need 
- A [GitHub](https://github.com/) Account 
- [VSCode](https://code.visualstudio.com/download) Installed
- [NodeJS](https://nodejs.org/en/download/) Installed
- [GoLang](https://golang.org/doc/install) Installed

### What You’ll Build 
- A Snowflake Guide

```
